android/app/src/main/res/layout/fragment_about.xml

```diff
@@ -19,7 +12,8 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:background="@color/white"
-            android:orientation="vertical">
+            android:orientation="vertical"
+            android:descendantFocusability="blocksDescendants">
```

Comment:
In case you're wondering about the use of this line.
If i don't add this line, then the home screen starts at the middle of the speakers recyclerview instead of being at the top of the layout.
[Here](https://stackoverflow.com/a/37288302/6748052) is the stackoverflow answer which suggests adding this.